style.css

```diff
@@ -0,0 +1,28 @@
+
+.container{
+    margin: 0 auto;
+    max-width: 1325px;    
+}
+
+
+header {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 15px;
+}
+
+
+.input-search{  
+     border-radius: 20px;
+     font-family: Muli-Regular;
+     font-size: 21px;
+     background: url(images/shape.png);
+     background-repeat: no-repeat;
+     background-position: center left 10px;
+     padding-left: 40px;
+     height: 40px; 
+}
+
+.logo{
+    height: 40px;
```

Comment:
We don't need to set the height to the logo, when you export the image from the sketch file the logo should have the correct size and we don't need to modify it.